apps/chat/src/store/conversations/conversations.epics.ts

```diff
@@ -1049,21 +1049,21 @@ const updateMessageEpic: AppEpic = (action$, state$) =>
         messages[payload.messageIndex].custom_content?.attachments;
 
       if (attachments) {
-        const attachmentFolders = uniq(
+        const attachmentRootFolders = uniq(
           attachments
             .map(
               (attachment) =>
                 attachment.url &&
-                getFolderIdFromEntityId(decodeURIComponent(attachment.url)),
+                getRootFolderIdFromEntityId(decodeURIComponent(attachment.url)),
```

Comment:
Use `getParentFolderIdsFromEntityId` instead to reset statuses for all parent hierarchy

Reply:
Thank you for the suggestion!
Changed to reset all parent hierarchy.